common/arg.cpp

```diff
@@ -1501,6 +1501,21 @@ common_params_context common_params_parser_init(common_params & params, llama_ex
             params.sampling.grammar = json_schema_to_grammar(json::parse(schema));
         }
     ).set_sparam());
+    add_opt(common_arg(
+        {"--gpu-sampling"},
+        "enable GPU sampling (default: disabled)",
+        [](common_params & params) {
+            params.sampling.gpu_sampling = true;
+        }
+    ).set_sparam());
+    add_opt(common_arg(
+        {"--gpu-dist"},
+        "perform final sampling on GPU (default: disabled)",
+        [](common_params & params) {
+            params.sampling.gpu_dist = true;
+            params.sampling.gpu_sampling = true;
+        }
+    ).set_sparam());
```

Comment:
I would have presumed `--gpu-sampling` to enable `--gpu-dist` as well; Maybe rename to something like "--gpu-sampling" and "--gpu-pre-sampling"? Or "--gpu-sampling" and "--hybrid-sampling"?